openedx/core/djangoapps/content/search/api.py

```diff
@@ -563,20 +577,93 @@ def upsert_library_block_index_doc(usage_key: UsageKey) -> None:
     _update_index_docs(docs)
 
 
+def _get_document_from_index(document_id: str) -> dict:
+    """
+    Returns the Document identified by the given ID, from the given index.
+
+    Returns None if the document or index do not exist.
+    """
+    client = _get_meilisearch_client()
+    document = None
+    try:
+        index = client.get_index(STUDIO_INDEX_NAME)
+        return index.get_document(id)
+    except (MeilisearchError, MeilisearchApiError) as err:
+        # The index or documennt doesn't exist
+        log.exception(err)
+
+    return None
```

Comment:
`document` is not used. I think you meant to save `index.get_document(id)` in `document` and return it at the end.
```suggestion
    document = None
    try:
        index = client.get_index(STUDIO_INDEX_NAME)
        document = index.get_document(id)
    except (MeilisearchError, MeilisearchApiError) as err:
        # The index or documennt doesn't exist
        log.exception(err)

    return document
```

Reply:
Oh wow, thank you for spotting those typos @navinkarkera ! Fixed with e92f40544e.